### PR TITLE
AutoGen Virtual Focus Group

### PR DESCRIPTION
**Pull request by msamylea**
## Why are these changes needed?

Added entry to gallery for AutoGen Virtual Focus Group

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: website/src/data/gallery.json [hunk 1]
```diff
@@ -1,11 +1,18 @@
-[
+f[
```
Comment (sonichi) [inline website/src/data/gallery.json:1]: Is the extra f a mistake?

Comment (skzhang1) [reply website/src/data/gallery.json:1]: yes

**Review by skzhang1 — APPROVED** [review]

File: website/src/data/gallery.json [hunk 2]
```diff
@@ -6,6 +6,13 @@
     "image": "autotx.png",
     "tags": ["tools", "groupchat", "app", "blockchain"]
   },
+  {
+    "title": "AutoGen Virtual Focus Group",
+    "link": "https://github.com/msamylea/autogen_focus_group",
+    "description": "A virtual consumer focus group with multiple custom personas, product details, and final analysis created with AutoGen, Ollama/Llama3, and Streamlit.",
+    "image": "default.png",
```
Comment (ekzhu) [inline website/src/data/gallery.json:13]: Would be great if you can provide an image (tiny size) for this entry.

**Review by rihp — CHANGES_REQUESTED** [review]

**Comment by krishnashed** [comment]
@msamylea we should be good to merge your PR after addressing comments added by @ekzhu @rihp

File: website/src/data/gallery.json (hunk 2 above)
Comment (sonichi) [reply website/src/data/gallery.json:13]: @msamylea if you'd like to provide an image for this entry, feel free to make another PR. I'm resolving it for now to merge this PR.

**Pull request merged**